tests/performance/order_by_single_nullable.xml

```diff
@@ -0,0 +1,14 @@
+<test>
+    <create_query>CREATE TABLE IF NOT EXISTS t_opt_orderby (a UInt64, b Nullable(UInt64)) ENGINE = Memory()</create_query>
+    <fill_query>insert into t_opt_orderby select * from  generateRandom('a UInt64, b Nullable(UInt64)') limit 10000000</fill_query>
```

Comment:
@taiyang-li I guess a performance tests for the low-cardinality optimization will also make sense.

Reply:
I think it was covered by sort.xml 
![image](https://github.com/user-attachments/assets/5f94fbf5-a562-4709-aff6-7bcb6738bf72)